_data/simple-icons.json

```
@@ -5225,6 +5225,20 @@
             "source": "https://gradle.com/brand",
             "guidelines": "https://gradle.com/brand"
         },
+        {
+            "title": "Gradle Play Publisher",
+            "hex": "82B816",
+            "source": "https://github.com/Triple-T/gradle-play-publisher/blob/df4eadf1ca6b5bad50e21be0b21816722ed50342/assets/logo.svg",
+            "license": {
+                "type": "MIT"
+            },
+            "aliases": {
+                "aka": [
+                    "gpp",
+                    "Triple-T"
+                ]
+            }    
```

Believe it or not, it's this that is causing the linter issue...

```suggestion
            }
```

---

Weirdly this issue was not showing on codespaces. I only did the change that I requested on the PR and the linter was passing.
Anyway `npm run add-icon-data` never fails

---

You are awesome! 👍